applications/tari_validator_node/src/bootstrap.rs

```diff
@@ -251,11 +252,20 @@ pub async fn spawn_services(
     }
     handles.push(epoch_manager_join_handle);
 
+    let validator_node_client_factory = TariValidatorNodeRpcClientFactory::new(networking.clone());
+
     info!(target: LOG_TARGET, "Template manager initializing");
     // Template manager
+    let (tx_template_sync, rx_template_sync) = broadcast::channel::<TemplateSyncRequest>(100);
```

Comment:
use an unbounded mpsc channel. Broadcast channel is not appropriate as it is typically used for events and if there are more than 100 requests some of them may start to be dropped (broadcast channel lagging)